### PR TITLE
Feature/vendor view testing

### PR DESCRIPTION
**Pull request by Devynnem**
____ Wrote Tests ____ Implemented ____ Reviewed

## Neccesary checkmarks
- [x] All Tests are Passing in all environments
- [x] The code will run locally
## Type of change
- [x] New feature
- [ ] Bug Fix
- [x] Refactor
## Description of Change:

  description: This will complete the testing for vendor view with all tests passing and being intercepted properly. This will also refactor a bit in vendor view so I could access the data properly with adding in a class name
    
## Requested feedback: 
    
  I'd like feedback on... 

## Check the correct boxes

- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes

- [ ] No Tests have been changed
- [x] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

- [ ] My code has no unused/commented out code
- [ ] My code has no binding.pry's
- [ ] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

**Review by Pma913 — APPROVED** [review]
Great job, this all looks great!

File: cypress/e2e/vendor_view_spec.cy.js [hunk 1]
```diff
@@ -1,5 +1,77 @@
...
+
+  beforeEach(() => {
+    cy.intercept("GET", "https://streeteatz-be-b15261620498.herokuapp.com/vendors/", {
+      statusCode: 200,
+      fixture: "truckData.json"
+    });
+    cy.intercept("GET", "https://streeteatz-be-b15261620498.herokuapp.com/vendors/9", {
+      statusCode: 200,
+      fixture: "singleTruckData.json"
+    })
```
Comment (JacobMacFarlane) [inline cypress/e2e/vendor_view_spec.cy.js:11]: Great work intercepting each and every endpoint!

File: src/Components/VendorView/VendorView.js [hunk 2]
```diff
@@ -44,8 +44,8 @@ const VendorView = ({ vendor, toggleLive, getAddress }) => {
             <div className="vv-left">
               <div className="statistics-container">
                 <h3>{currentVendor.name} Statistics!</h3>
-                <p>You have a total of {currentVendor.up_rating} Streeteatz users who love your truck!</p>
-                <p>You have a total of {currentVendor.down_rating} Streeteatz users who love your truck!</p>
+                <p className='statistics-info'>You have a total of {currentVendor.up_rating} Streeteatz users who love your truck!</p>
+                <p className='statistics-info'>You have a total of {currentVendor.down_rating} Streeteatz users who need more convincing to love your truck!</p>
```
Comment (JacobMacFarlane) [inline src/Components/VendorView/VendorView.js:48]: Added this for testing because i forgot it when i made it im assuming!

**Review by JacobMacFarlane — COMMENTED** [review]
Great Work!

**Review by Pma913 — APPROVED** [review]

**Pull request merged**